start_ocean.sh

```diff
@@ -45,12 +58,15 @@ while :; do
             printf $COLOR_Y'Starting and reusing the database ...\n\n'$COLOR_RESET
             ;;
         --no-pleuston)
-            COMPOSE_FILE='docker-compose-no-pleuston.yml'
+            COMPOSE_FILES=${COMPOSE_FILES/ -f ${COMPOSE_DIR}\/pleuston.yml/}
             printf $COLOR_Y'Starting without Pleuston...\n\n'$COLOR_RESET
             ;;
         --local-parity-node)
```

Comment:
what is this starting? a new network or a node that connects to the testnet?

Reply:
It depends the variable `KEEPER_NETWORK_NAME`. With this refactor the start_ocean.sh should work exactly the same as before the refactor (there is not any functionality change).
That line in particular remove the `pleuston.yml` docker-compose in the running sequence.